symphony-bdk-core/src/test/java/com/symphony/bdk/core/service/message/MessageServiceTest.java

```diff
@@ -441,6 +444,16 @@ void testDoSend(final BdkMockServer mockServer) throws IOException, ApiException
     assertInvokeApiCalledWithCorrectParams(mockServer, message, Collections.emptyList(), Collections.emptyList());
   }
 
+  @Test
+  void testDoSendLimitExceededMessage() throws ApiException {
+    doThrow(new ApiException(400, "Your message exceeds the maximum number of characters allowed."))
+        .when(agentClient).invokeAPI(any(), any(), any(), any(), any(), any(), any(), any(), any(), any(), any());
+
+    String message = "<messageML>" + StringUtils.repeat('a', 70000) + "</messageML>";
```

Comment:
we don't need such a big message for the test :)

Reply:
I just want to make the ut looks realistic :)